### DISCUSSION TITLE
Increase the usage of Discussions

### DISCUSSION THREAD

**Discussion by pavelkeyzik** [primer/prism#30] [Done]
As we can see on the issues page we have items that actually some ideas and suggestions. What do you think is possible to do to increase the amount of new discussion, instead of creating new issues with ideas?

I guess the only way to do this for now will be to create `config.yml` file in `ISSUE_TEMPLATE` folder that will add another item called "Feature Request/Question" that will have a link to open Discussion. At the moment, when users click "New Issue" button, they see "Bug report" and "Don’t see your issue here? Open a blank issue" and if it's just idea, of course they will click the second item.

<img width="719" alt="CleanShot 2022-06-19 at 10 24 17@2x" src="https://user-images.githubusercontent.com/17102399/174470389-86589460-4714-4148-b4b5-c92d9b7de2c6.png">

Does anyone have other ideas how to do this?

**Comment by colebemis** [comment]
Love this idea! Want to open a PR?

**Reply by pavelkeyzik** [reply]
Of course 😄

**Reply by pavelkeyzik** [reply]
Any suggestions about the copy? I will need title and description

**Comment by pavelkeyzik** [comment]
I've opened a [PR](https://github.com/primer/prism/pull/34). Any suggestions are welcome 😊

**Comment by pavelkeyzik** [comment]
@colebemis Is it possible to mark discussion as "Implemented/Resolved"? Can't check on the laptop at the moment 😊

**Reply by colebemis** [reply]
Great question. I was just thinking about this. I think we can convert them to an issue and close them as "done" once they're implemented.

**Reply by colebemis** [reply]
I'll give that a try with this discussion

**Reply by colebemis** [reply]
Ah, looks like this is an open issue: https://github.com/github-community/community/discussions/3097

For now, I'll create a `done` label that we can add to ideas that have been implemented

**Reply by pavelkeyzik** [reply]
Hmm.. I see they have a label "Pull Request" and category with the same name. What if we just create a category "Done" and just move all discussion to this category. So, we'll get workflow like "Idea" -> "Done". We can also add some steps in between. That way we don't need to filter all ideas by labels 😊

**Reply by colebemis** [reply]
Great idea. Let's try that
